datafusion/physical-expr/src/aggregate/sum.rs

```diff
@@ -236,15 +236,15 @@ impl Accumulator for SumAccumulator {
 
     fn update_batch(&mut self, values: &[ArrayRef]) -> Result<()> {
         let values = &values[0];
-        self.count += (values.len() - values.data().null_count()) as u64;
+        self.count += (values.len() - values.null_count()) as u64;
```

Comment:
This results in a test failure, as it reports the arguably correct null count, instead of 0 for NullArray. I'm not sure if this is a bug or not...